test/e2e/common/05_tektonhubdeployment_test.go

```diff
@@ -94,6 +94,12 @@ func NewTektonHubTestSuite(t *testing.T) *TektonHubTestSuite {
 // before suite
 func (s *TektonHubTestSuite) SetupSuite() {
 	resources.PrintClusterInformation(s.logger, s.resourceNames)
+
+	// reset the tekton config into default state
+	s.logger.Debug("resetting TektonConfig to it's default state")
+	tcSuite := NewTestTektonConfigTestSuite(s.T())
+	tcSuite.recreateOperatorPod()
+	tcSuite.resetToDefaults()
```

Comment:
Is this change related to your pr 
May be I think you need to rebase your pr with main